src/pages/Polls.jsx

```diff
@@ -13,12 +13,23 @@ const Polls = (props) => {
   const [isPollsLoading, setIsPollsLoading] = useState(false);
 
   useEffect(() => {
-    getPolls();
+    if(props.userType == 'respondent'){
+      getRespondentPolls();
+    } else{
+      getManagerPolls();
+    }
   }, []);
 
-  async function getPolls() {
+  async function getRespondentPolls() {
     setIsPollsLoading(true);
-    const response = await PollService.getUserPolls(props.statuses);
+    const response = await PollService.getRespondentPolls(props.statuses);
```

Comment:
можно оставить одну функцию getPolls
```suggestion
    const response = props.userType === 'respondent' ? await PollService.getRespondentPolls(props.statuses) : await PollService.getManagerPolls();
```